docs/cache/develop/authentication/api-keys.md

```diff
@@ -16,13 +16,13 @@ API keys are *long-lived values intended for programmatic use*. These keys grant
 
 :::info
 
-You are not able to create "super-user" API keys via the Momento SDK. However, you *can* create fine-grain access controlled keys! Check out our [Auth API reference page](./../api-reference/auth.md) for more details.
+It is not possible to create "super-user" API keys via the Momento SDK; these may only be created in the console. However, you *can* use the SDK to create API keys with specific permissions, via fine-grain access control! Check out our [Auth API reference page](./../api-reference/auth.md) for more details.
 
 :::
 
 ## Creating an API key
 
-While you are certainly allowed to create API keys via our SDK, it is generally recommended to use the [Momento console](https://console.gomomento.com/tokens). This allows you to monitor and maintain your long-lived keys visually, making sure you don't accidentally open up a security hole in your account. 
+While it is possible to create API keys via the Momento SDK, it is generally recommended to use the [Momento console](https://console.gomomento.com/tokens). This allows you to monitor and maintain your long-lived keys visually, making sure you don't accidentally open up a security hole in your account. 
```

Comment:
This gives me the impression that I can see a list of all my api keys on the console - is that something we implemented on the console? Because afaik, we don't store anything in the service.

Reply:
yeah, you're correct, we don't. I didn't actually edit that part but it seems like I should :) will add a commit.

Reply:
@poppoerika can you weigh in on this? I want to make sure that whatever changes we make will address the confusion you were having the other day when you brought this to my attention.

Reply:
resolved in https://github.com/momentohq/public-dev-docs/pull/473/commits/da4c69c2922f3697b3af7721aad3183a0ad4d215